### DISCUSSION TITLE
长列表上滑加载，setData 数据传输长度为 1764 KB，存在有性能问题！

### DISCUSSION THREAD

**Discussion by wsaisai** [NervJS/taro#14798] [Q & A]
用taro写的评价页面，上滑加载数据 每次进行concat  最后使用setState方法进行更新，现在的情况是加载超过1024之后就白屏了，请问怎么解决？
  useReachBottom(async () => {
    const { currentPage, appraiseList, totalElement } = state;
    // eslint-disable-next-line @typescript-eslint/no-unused-vars
    let page = currentPage;
    Taro.showLoading({
      title: '加载中',
    });
    if (appraiseList.length >= totalElement) {
      Taro.hideLoading();
      setState({
        moreData: true,
      });
    } else {
      page++;
      const list = appraiseList;
      const result: any = await getAppraise(page);
      const currentList: any = list.concat(result);
      setState({
        currentPage: page,
        appraiseList: currentList,
      });
      if (currentList) {
        setTimeout(() => {
          Taro.hideLoading();
        }, 1000);
      }
    }
  });

**Comment by Chen-jj** [comment]
@wsaisai 麻烦提一下 issue 并附上可复现的 demo
